components/clarinet-sdk/tests/index.test.ts

```diff
@@ -314,3 +277,14 @@ describe("simnet can get session reports", async () => {
     expect(report.cost_result.total.write_count).toBe(3);
   });
 });
+
+describe("the sdk handles multiple manifests project", () => {
+  it("handle invalid project", () => {
+    const expectedErr =
+      "error: unexpected ')'\n--> /Users/hugo/Sites/hiro/clarinet/components/clarinet-sdk/tests/fixtures/contracts/invalid.clar:5:2\n)) ;; extra `)`\n";
```

Comment:
> /Users/hugo
Will this test pass on other people's computers?

Reply:
Also related to test above, when running it, I am receiving an `AssertionError: promise resolved "undefined" instead of rejecting` error.

Reply:
I mean, it runs on my machine 🤦‍♂️ 
Thanks for catching it @MicaiahReid 
I'll double check @csgui 👍

Reply:
@csgui 
Ah yes, it can't be run out of the box. 

You first need build `clarinet-sdk-wasm`  with `wasm-pack build --release --target=nodejs --scope hirosystems` (see the README) then run `npm link` in the generated package
In `clarinet-sdk`, run `npm link @hirosystems/clarity-sdk-wasm`.

You only need to run `npm link` once even if the wasm is re-built.


I really need to find a better development flow for that (or at least document it until then).